Makefile

```diff
@@ -246,6 +246,13 @@ verify-crd:
 	$(diff) '$(tmp_file)' '$(CRD_PATH)' || (echo 'File $(CRD_PATH) is not up-to date. Please run `make update-crd` to update it.' && false)
 .PHONY: verify-crd
 
+# $1 - Helm charts dir
+define update-helm-schemas
+	@crd_schema=$$( $(YQ) eval -j '.spec.versions[0].schema.openAPIV3Schema | del(..|.description?)' $(CRD_PATH) ); \
```

Comment:
What happens if we add v2alpha1 to the CRD? I guess you should delete the description in all versions (and select the correct one) and it may not be enough in the future still. yq is missing `--argfile` argument so the env will eventually grow over the system limit to pass through. Funny enough I was swearing at that today when generating OLM manifests but I got away with higher reduction (`$(YQ) eval 'del(.spec.versions[].schema.openAPIV3Schema.properties)`).

Reply:
Changed to `(.spec.versions[] | select(.name == "v1")`. This is UB (well, it will most likely generate an invalid json) in case there is two versions with the same name. Figured this is better than using `any` (or any other way to do get the same result in `yq` as `any` is missing). The target is now generating the entire file and not using variables.